src/ARM.h

```diff
@@ -322,6 +322,7 @@ class ARMv5 : public ARM
     u32 RNGSeed;
 
     u32 DTCMSetting, ITCMSetting;
+    u32 DCacheLockDown, ICacheLockDown;
```

Comment:
Don't forget to add these to the savestate (and to increment the savestate version).